Makefile

```diff
@@ -45,11 +57,30 @@ ios:
 	@echo "Import \"$(GOBIN)/Geth.framework\" to use the library."
 
 test:
-	# Skip mobile and cmd tests since they are being deprecated
-	go test -v $$(go list ./... | grep -v go-ethereum/cmd/) -cover -coverprofile=cover.out
+	$(GOTEST) --timeout 5m -shuffle=on -cover -coverprofile=cover.out $(TESTALL)
```

Comment:
What's the purpose?

Reply:
Since we will fail CI when linter fails anyway, I think it may be more convenient to combine unit tests and linter into one command. I am also fine with separating them. It is your call. :D

Reply:
It should we 'green' now.